umbraco/Umbraco.CMS.Backoffice
# Update to Node 18

## Changes
* This bumps our support to Node 18, which seemingly runs without issues.
* Also added `.nvmrc` file to instruct NVM which version to use
 
## How to test
1. Make sure you have Node 18.14 installed (the latest LTS as of writing).
2. Go to the terminal and write `nvm use` and see that it loads the correct version (18.14 as of this writing)
3. Test `npm run dev`
4. Test `npm test`
